docs/examples/cloud_trace_exporter/README.rst

```diff
@@ -47,4 +49,33 @@ Running basic_trace.py hangs:
 
 Getting error ``google.api_core.exceptions.ResourceExhausted: 429 Resource has been exhausted``:
 ################################################################################################
-    * Check that you've enabled the `Cloud Trace (Stackdriver Trace) API <https://console.cloud.google.com/apis/library?q=cloud_trace>`_
\ No newline at end of file
+    * Check that you've enabled the `Cloud Trace (Stackdriver Trace) API <https://console.cloud.google.com/apis/library?q=cloud%20trace&filter=visibility:public>`_
+
+bash: pip: command not found:
```

Comment:
Kind of surprising to see this since not everyone uses bash, but I think it's fine to leave as is since most people probably are.